packages/components/pages/f-checkout/CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v4.6.1
+------------------------------
+*September 28, 2022*
+
+### Added
+- New props to README
```

Comment:
Please could you mention the names of the new props here? It makes it a bit easier when you're skimming the changelog